### ISSUE TITLE
levelPaths ignored when importing OGMO project

### ISSUE THREAD

**Issue by jstolarek** [deepnight/ldtk#425]
I keep getting an error when attempting to import an OGMO project. Here's the structure of my `res/` directory:

```
res/
|
|--- testlevels.ogmo
|
|--- foo.json
|
|--- levels/
     |
     |--- level_1.json
     |
     |--- level_2.json
```

`foo.json` is unrelated to OGMO project - it's an Aseprite export. Here's a fragment of the ogmo file:

```
{
  "name": "Test levels",
  "ogmoVersion": "3.3.0",
  "levelPaths": ["levels/"],
...
```

When I try to import `testlevels.ogmo` I get an error and the log shows LDtk attempts to read `foo.json` and fails because it doesn't have expected structure. It looks like `levelPaths` entry is ignored during import. See [here](https://github.com/deepnight/ldtk/blob/f9bb12fdaf90deddf752d0ba96787778c720affc/src/electron.renderer/importer/OgmoProject.hx#L265)

**Comment by deepnight** [comment]
Thank you, I'll check this out :)

**Comment by deepnight** [comment]
Fixed in 0.9.2
